front/app/components/SignUpIn/AuthProviderButton.tsx

```diff
@@ -154,7 +154,7 @@ const AuthProviderButton = memo<Props>(
     );
 
     const handleContinueClicked = useCallback(() => {
-      if (!tacAccepted) {
+      if (!tacAccepted && authProvider !== 'id_vienna_saml') {
```

Comment:
Is there no way to avoid this conditional logic in our generalized component? To me, this is a sign that perhaps we shouldn't use this component but create a specific one for vienna in the module itself.

Reply:
I'm happy to follow up on this, but I won't be able to turn around on my own until EOD today, sorry. :[ I agree that the conditional special vienna logic makes little sense to me, but someone needs to guide me how to refactor this out of the module.
I'm specifically unsure how to deal with shared behaviour, styling of these two components. Is it okay to live the duplication or should they have a common ancestor?